packages/react/src/components/builder-content.component.tsx

```diff
@@ -45,6 +45,12 @@ export type BuilderContentProps<ContentType> = {
   content?: BuilderContent;
 } & ({ model: string } | { modelName: string }); // model and modelName are aliases of the same thing¸
 
+/**
+ * When passed content json explicitly it'll calculate a/b tests on the content
+ * and pass the winning variation down to the children function. If then content
+ * prop was omitted it'll try to fetch matching content from your Builder
+ * account based on the default user attributes and model.
+ */
```

Comment:
Maybe worth the disclaimer that test selection, etc happens only if DoNotTrack is false (which in that case, only the default variation is used)

Reply:
Done